system/networking.md

```diff
@@ -19,6 +19,11 @@ In Qubes, the standard Xen networking is used, based on backend driver in the dr
 
 The IP address of `eth0` interface in AppVM, as well as two IP addresses to be used as nameservers (`DNS1` and `DNS2`), are passed via xenstore to AppVM during its boot (thus, there is no need for DHCP daemon in the network driver domain). `DNS1` and `DNS2` are private addresses; whenever an interface is brought up in the network driver domain, the */usr/lib/qubes/qubes\_setup\_dnat\_to\_ns* script sets up the DNAT iptables rules translating `DNS1` and `DNS2` to the newly learned real dns servers. This way AppVM networking configuration does not need to be changed when configuration in the network driver domain changes (e.g. user switches to a different WLAN). Moreover, in the network driver domain, there is no DNS server either, and consequently there are no ports open to the VMs.
 
+Cooperative Covert Channels Between Domains
+-------------------------------------------
+
+Qubes does not attempt to eliminate cooperative covert channels between two or more compromised domains. We don't believe this is possible to achieve on x86 hardware, and we also doubt it makes any sense in practice for most users -- after all, if the two domains are compromised, then it's already (almost) all lost anyway.
```

Comment:
1. Which "other article"?
2. Just because another article says it doesn't make it true. If the other article is wrong, we shouldn't transfer the wrongness into other documents.